CHANGES.md

```diff
@@ -31,8 +31,13 @@ Please follow the [`CHANGES.md` modification guidelines](https://github.com/dcos
 * Port the Mesos open source metrics module to Windows. (DCOS-58008)
 
 * Add etcd into DC/OS. (DCOS-59004)
+
 * Update libpq to 9.6.15 (DCOS-59145)
 
+* Calico in DC/OS: introduced Calico network for mesos UCR. (DCOS-58582)
```

Comment:
We can replace these multiple changes by just one item, `introducing calico in DC/OS`, added a TODO here: https://github.com/mainred/dcos/pull/2/files#r356544985